scene/resources/curve.h

```diff
@@ -302,6 +303,12 @@ class Curve3D : public Resource {
 	static void _bind_methods();
 
 public:
+#ifdef TOOLS_ENABLED
+	// For Path3DGizmo.
+	mutable Vector<size_t> points_in_cache;
```

Comment:
Keep points_in_cache private. The method can be public, but such data should basically be private.

Reply:
done